crates/trigger-redis/src/spin.rs

```diff
@@ -11,6 +12,7 @@ pub struct SpinRedisExecutor;
 
 #[async_trait]
 impl RedisExecutor for SpinRedisExecutor {
+    #[instrument(name = "spin_trigger_redis.execute", skip(self, engine, payload), err(level = Level::INFO), fields(otel.name = format!("execute_wasm_component {}", component_id)))]
```

Comment:
```suggestion
    #[instrument(name = "spin_trigger_redis.execute_wasm", skip(self, engine, payload), err(level = Level::INFO), fields(otel.name = format!("execute_wasm_component {}", component_id)))]
```

^nit for consistency